pkg/cloudprovider/gce/gce.go

```diff
@@ -251,18 +273,17 @@ func (gce *GCECloud) CreateTCPLoadBalancer(name, region string, externalIP net.I
 		Name:       name,
 		IPProtocol: "TCP",
 		PortRange:  fmt.Sprintf("%d-%d", minPort, maxPort),
-		Target:     pool,
```

Comment:
It's existing code, but I couldn't help noticing that the above for loop will end up creating a single forwarding rule spanning everything from minPort to maxPort (I think).  e.g. if ports[0]==1 and ports[1]==65000 we'll end up forwarding 65000 ports, and not just 2.  Am I right?